doc/source/scans/tutorial.rst

```diff
@@ -302,6 +311,25 @@ Perform Fits
   .. figure:: peak.png
      :alt: Fitting a peak
 
+  An alternative 'fitting' method is a "centre of mass" fit. For a set of
+  data points (x, y) it calculates the centre of mass as sum(x*y)/sum(y).
+  The background is subtracted before this calculation is done.
+
+  >>> fit = scan(theta, start=0, stop=2, count=11, fit=CentreOfMass, frames=5, save="centre_of_mass.png")
```

Comment:
I'm getting an error on this line in the test suite.  The full stack trace is:

    Failed example:
        fit = scan(theta, start=0, stop=2, count=11, fit=CentreOfMass, frames=5, save="centre_of_mass.png")
    Exception raised:
        Traceback (most recent call last):
          File "/nix/store/9whbvdrw03qzwcqgk4zjv31rqdahv677-python-2.7.15-env/lib/python2.7/doctest.py", line 1315, in __run
            compileflags, 1) in test.globs
          File "<doctest tutorial.rst[50]>", line 1, in <module>
            fit = scan(theta, start=0, stop=2, count=11, fit=CentreOfMass, frames=5, save="centre_of_mass.png")
          File "/home/adam/Code/InstrumentScripts/general/scans/util.py", line 106, in inner
            return getattr(obj, method)(*args, **kwargs)
          File "/home/adam/Code/InstrumentScripts/general/scans/defaults.py", line 148, in scan
            return scn.fit(**kwargs)
          File "/home/adam/Code/InstrumentScripts/general/scans/scans.py", line 235, in fit
            result = self.plot(action=fit.fit_plot_action(), **kwargs)
          File "/home/adam/Code/InstrumentScripts/general/scans/scans.py", line 199, in plot
            axis)
          File "/home/adam/Code/InstrumentScripts/general/scans/fit.py", line 537, in action
            params = self.fit(x, y)
          File "/home/adam/Code/InstrumentScripts/general/scans/fit.py", line 497, in fit
            interpolated_x = np.array(np.arange(np.min(x), np.max(x), float(np.max(x) - np.min(x))/len(raw_data)))
        ValueError: arange: cannot compute length